sdk/communication/azure-mgmt-communication/CHANGELOG.md

```diff
@@ -1,5 +1,5 @@
 # Release History
 
-## 1.0.0b1 (2020-09-22)
+## 0.1.0 (1970-01-01)
```

Comment:
this should be align with version in _version.py

Reply:
Should we keep the 1.0.0b1 (2020-09-22) in the history? As in, have two entries here.